src/core/Object3D.js

```diff
@@ -320,13 +320,23 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 			}
 
+			const position = this.parent ? this.position : this.getWorldPosition( worldPosition );
```

Comment:
The order seems incorrect. If `this.parent` is not `null`, we want to use `Object3D.getWorldPosition()`. Otherwise `this.position` is already in world space.

Reply:
Oh, the code in your fiddle is actually correct^^.

Reply:
Doh! I thought I fixed that before checking in 😅